### PR TITLE
Add GitHub action to perform pull request labelling.

### PR DESCRIPTION
**Pull request by gitaumoses4**
## What does this PR do?

- Adds an automatic labeler to pull requests based on modified paths.

## More Info
The automatic labeler action can be found [here](https://github.com/marketplace/actions/labeler)

I made a tiny change on the `msalautomationapp/` project to check whether it will trigger the addition of a  `test-automation` label to this pull request.

### CODE REVIEW

File: msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/MainActivity.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
+
```
Comment (gitaumoses4) [inline msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/MainActivity.java:23]: This here is to trigger the `test-automation` label.

**Review by shahzaibj — APPROVED** [review]

**Review by iambmelt — APPROVED** [review]
LGTM, thanks @gitaumoses4!

**Comment by iambmelt** [comment]
(I took the liberty of merging the latest from `dev` - hope you don't mind 😉 )

File: msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/MainActivity.java (hunk 1 above)
Comment (iambmelt) [reply msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/MainActivity.java:23]: Feel free to revert this before you commit. I believe you can use `git commit --allow-empty -m "Empty commit"` to trigger builds, tag-events, etc in the future.

Cheers!

Comment (gitaumoses4) [reply msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/MainActivity.java:23]: Thanks for this tip @iambmelt.

**Pull request merged**